src/MetaModels/Attribute/Checkbox/Checkbox.php

```diff
@@ -93,35 +93,19 @@ public function getFieldDefinition($arrOverrides = array())
      */
     public function getFilterOptions($idList, $usedOnly, &$arrCount = null)
     {
-        $values = parent::getFilterOptions($idList, $usedOnly, $arrCount);
-        // If not used only or id list, we need to override the handling now.
-        if (!($idList || $usedOnly)) {
-            $values = ['', '1'];
-            if (null !== $arrCount) {
-                $arrCount[''] = 0;
-                $arrCount[1]  = 0;
-                $colName      = $this->getColName();
-                $rows         = $this->getDatabase()->execute(
-                    sprintf(
-                        'SELECT %1$s, COUNT(%1$s) as mm_count FROM %2$s GROUP BY %1$s ORDER BY %1$s',
-                        $colName,
-                        $this->getMetaModel()->getTableName()
-                    )
-                );
-                while ($rows->next()) {
-                    $arrCount[$rows->$colName] = $rows->mm_count;
-                }
-            }
+        
+        $arrOptions = parent::getFilterOptions($idList, $usedOnly, $arrCount);
+        foreach ($arrOptions as $key => $value) {
+           if ($key == '1') {
+              $arrOptions['1'] = $GLOBALS['TL_LANG']['MSC']['metamodelattribute_checkbox']['value_1'];
+           }else{
+            unset($arrOptions[$key]);
+            $arrOptions['0'] = $GLOBALS['TL_LANG']['MSC']['metamodelattribute_checkbox']['value_0'];
+           }
         }
+           
+        return $arrOptions;
 
-        // Finally use the correct language strings.
```

Comment:
Although the second question rather belongs to the code block [above](https://github.com/MetaModels/attribute_checkbox/pull/18/files#r106331456) I'll answer here, as the question is here.

We are not counting the amount of options but the amount of ENTRIES per option.
Like: `['' => 450, 1 => 50]` which means that there are 450 items matching which are having a value of `''` and 50 items having a value of `1`.